ReactAndroid/src/main/jni/third-party/glog/Android.mk

```diff
@@ -3,24 +3,24 @@ LOCAL_PATH:= $(call my-dir)
 include $(CLEAR_VARS)
 
 LOCAL_SRC_FILES := \
-  glog-0.3.3/src/demangle.cc \
-  glog-0.3.3/src/logging.cc \
-  glog-0.3.3/src/raw_logging.cc \
-  glog-0.3.3/src/signalhandler.cc \
-  glog-0.3.3/src/symbolize.cc \
-  glog-0.3.3/src/utilities.cc \
-  glog-0.3.3/src/vlog_is_on.cc
+  glog-0.3.5/src/demangle.cc \
+  glog-0.3.5/src/logging.cc \
+  glog-0.3.5/src/raw_logging.cc \
+  glog-0.3.5/src/signalhandler.cc \
+  glog-0.3.5/src/symbolize.cc \
+  glog-0.3.5/src/utilities.cc \
+  glog-0.3.5/src/vlog_is_on.cc
 
-LOCAL_C_INCLUDES += $(LOCAL_PATH) $(LOCAL_PATH)/.. $(LOCAL_PATH)/glog-0.3.3/src/
+LOCAL_C_INCLUDES += $(LOCAL_PATH) $(LOCAL_PATH)/.. $(LOCAL_PATH)/glog-0.3.5/src/
 
-LOCAL_EXPORT_C_INCLUDES := $(LOCAL_PATH)/.. $(LOCAL_PATH)/glog-0.3.3/src/
+LOCAL_EXPORT_C_INCLUDES := $(LOCAL_PATH)/.. $(LOCAL_PATH)/glog-0.3.5/src/
 
 LOCAL_CFLAGS += \
   -Wall \
   -Wwrite-strings \
   -Woverloaded-virtual \
-  -Wno-sign-compare \
```

Comment:
You shouldn't add this suppress.
The lib had a bug:
https://github.com/google/glog/commit/a5ffa884137f7687d0393ccba22557d583654a25